params.yml

```diff
@@ -53,6 +38,9 @@ args:
   # In the future, we would like to support external DA protocols such as Avail, Celestia and Near.
   data_availability_mode: cdk-validium
 
+  # Additional services to run alongside the network.
+  additional_services: []
```

Comment:
I think you should have all possible additional services commented here. Would be helpful.

Reply:
👍 addressed in [docs: list additional services options in params.yml](https://github.com/0xPolygon/kurtosis-cdk/pull/230/commits/903d72c682f3ec44f24f1cac0181130302a7fda7)